### DISCUSSION TITLE
Allegato Lezione

### DISCUSSION THREAD

**Discussion by Forrest-It** [iisgiua/giuaschool#418] [Richieste di aiuto] [answered]
Volevo chiedere se c'è modo di inserire un allegato per una lezione, e/o se c'è un modo di legare alla lezione un documento .pdf

Grazie

**Comment by trinko** [comment, marked as answer]
Inserire un allegato no, ma si può utilizzare una soluzione equivalente:
- inserire nell'argomento l'indirizzo web della risorsa (i link sono cliccabili); si può ad esempio inserire il PDF su Google Drive e poi inserire il link di condivisione negli argomenti della lezione

**Comment by Forrest-It** [comment]
Buongiorno grazie,

una domanda : nella bacheca - vista Genitore - c'è una sezione dedicata ai documenti : come si alimentano ? chi e come si possono inserire documenti destinati ai genitori ? 
Potrebbe essere una soluzione più semplice da adottare : i docenti potrebbero inserire nel testo della sezione un riferimento a un documento pubblicato in bachca.

Grazie

Tommaso

**Comment by trinko** [comment]
Attualmente i documenti pubblicati nella sezione BACHECA > DOCUMENTI sono relativi a quelli dei programmi svolti e del documento del 15 maggio, inseriti tramite la gestione del menu DOCUMENTI.
Inoltre vengono resi visibili ai soli docenti del consiglio di classe altri documenti, come ad esempio quelli per gli alunni BES (PEI, PDP, ecc.)

Deve essere ancora implementata la possibilità di inserire e rendere visibile a diversi destinatori, altri tipi di documenti.
